docs/core-extensions.md

```diff
@@ -0,0 +1,68 @@
+# Extensions to Existing Ruby Classes
```

Comment:
@knowuh did you mean to add this file?  It was added in the commit where you removed `RedCloth`

Reply:
Yeah -- I converted a textile document to markdown and committed it. I didn't look at the file too closely, it's probably outdated, and could probably be removed.